config/prod/orca-vpc.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.6.8/templates/VPC/vpc-20-private.yaml"
```

Comment:
could we use [vpc-mini.yaml](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/VPC/vpc-mini.yaml) instead?

it would be `{{stack_group_config.aws_infra_templates_root_url}}/v0.6.8/templates/VPC/vpc-mini.yaml`

Reply:
@thomasyu888 @BrunoGrandePhD Are we good on using the mini vpc for now?

Reply:
Yeah, I indicated that we can use a mini VPC [here](https://sagebionetworks.slack.com/archives/CEFQD0KU1/p1678478990492689?thread_ts=1677689599.260229&cid=CEFQD0KU1). My response was a day late because I was OOO yesterday. Sorry about that.,